.gitbook.yaml

```diff
@@ -0,0 +1,7 @@
+# Do not edit this file, to adjust the table of contents, modify SUMMARY.md
+
+root: ./tutorial/
+
+structure:
+  readme: README.md
+  summary: SUMMARY.md
```

Comment:
Out of interest, what is the reason for introduction the gitbook file?

Reply:
so we use these labs in gitbook format! https://ibm-developer.gitbook.io/helm101/

Reply:
Ah, nice!